wal/src/table_kv_impl/table_unit.rs

```diff
@@ -996,13 +996,6 @@ impl TableUnitWriter {
             table_unit_state.table_id, sequence_num, table_unit_meta_table
         );
 
-        ensure!(
```

Comment:
Why remove this check?

Reply:
- This check is useless here, sequence could only advance in put. 
- We use MAX_SEQUENCE_NUMBER to represent delete all.